--- glslang-807a0d9e2f4e176f75d62ac3c179c81800ec2608/glslang/CMakeLists.txt.orig	2016-12-22 01:40:29.000000000 +0100
+++ glslang-807a0d9e2f4e176f75d62ac3c179c81800ec2608/glslang/CMakeLists.txt	2017-05-26 19:10:05.532591364 +0200
@@ -92,4 +92,4 @@
 endif(WIN32)
 
 install(TARGETS glslang 
-        ARCHIVE DESTINATION lib)
+        ARCHIVE DESTINATION lib${LIB_SUFFIX})
--- glslang-807a0d9e2f4e176f75d62ac3c179c81800ec2608/OGLCompilersDLL/CMakeLists.txt.orig	2016-12-22 01:40:29.000000000 +0100
+++ glslang-807a0d9e2f4e176f75d62ac3c179c81800ec2608/OGLCompilersDLL/CMakeLists.txt	2017-05-26 19:10:19.595924534 +0200
@@ -8,4 +8,4 @@
 endif(WIN32)
 
 install(TARGETS OGLCompiler 
-        ARCHIVE DESTINATION lib)
+        ARCHIVE DESTINATION lib${LIB_SUFFIX})
--- glslang-807a0d9e2f4e176f75d62ac3c179c81800ec2608/SPIRV/CMakeLists.txt.orig	2016-12-22 01:40:29.000000000 +0100
+++ glslang-807a0d9e2f4e176f75d62ac3c179c81800ec2608/SPIRV/CMakeLists.txt	2017-05-26 19:10:32.555924389 +0200
@@ -50,4 +50,4 @@
 endif(WIN32)
 
 install(TARGETS SPIRV SPVRemapper
-        ARCHIVE DESTINATION lib)
+        ARCHIVE DESTINATION lib${LIB_SUFFIX})
--- glslang-807a0d9e2f4e176f75d62ac3c179c81800ec2608/hlsl/CMakeLists.txt.orig	2016-12-22 01:40:29.000000000 +0100
+++ glslang-807a0d9e2f4e176f75d62ac3c179c81800ec2608/hlsl/CMakeLists.txt	2017-05-26 19:11:24.195923799 +0200
@@ -25,4 +25,4 @@
 endif(WIN32)
 
 install(TARGETS HLSL
-        ARCHIVE DESTINATION lib)
+        ARCHIVE DESTINATION lib${LIB_SUFFIX})
--- glslang-807a0d9e2f4e176f75d62ac3c179c81800ec2608/glslang/OSDependent/Unix/CMakeLists.txt.orig	2016-12-22 01:40:29.000000000 +0100
+++ glslang-807a0d9e2f4e176f75d62ac3c179c81800ec2608/glslang/OSDependent/Unix/CMakeLists.txt	2017-05-26 19:14:17.379255155 +0200
@@ -2,4 +2,4 @@
 set_property(TARGET OSDependent PROPERTY FOLDER glslang)
 
 install(TARGETS OSDependent 
-        ARCHIVE DESTINATION lib)
+	ARCHIVE DESTINATION lib${LIB_SUFFIX})
