--- glslang-11.1.0/CMakeLists.txt.orig	2021-04-28 20:23:56.849255577 +0200
+++ glslang-11.1.0/CMakeLists.txt	2021-04-28 20:25:03.082717552 +0200
@@ -320,10 +320,6 @@ if(BUILD_EXTERNAL AND IS_DIRECTORY ${CMA
     add_subdirectory(External)
 endif()
 
-if(NOT TARGET SPIRV-Tools-opt)
-    set(ENABLE_OPT OFF)
-endif()
-
 if(ENABLE_OPT)
     message(STATUS "optimizer enabled")
     add_definitions(-DENABLE_OPT=1)
--- glslang-11.1.0/SPIRV/CMakeLists.txt.orig	2020-12-08 02:20:27.000000000 +0100
+++ glslang-11.1.0/SPIRV/CMakeLists.txt	2021-04-28 18:11:13.736491495 +0200
@@ -95,7 +95,7 @@
         PRIVATE ${spirv-tools_SOURCE_DIR}/include
         PRIVATE ${spirv-tools_SOURCE_DIR}/source
     )
-    target_link_libraries(SPIRV PRIVATE MachineIndependent SPIRV-Tools-opt)
+    target_link_libraries(SPIRV PRIVATE MachineIndependent SPIRV-Tools-opt SPIRV-Tools)
     target_include_directories(SPIRV PUBLIC
         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../External>
         $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/External>)
