--- glslang-7.11.3214/CMakeLists.txt.orig	2019-05-09 10:11:28.000000000 +0200
+++ glslang-7.11.3214/CMakeLists.txt	2019-05-30 22:17:18.543918193 +0200
@@ -117,10 +117,6 @@
 # We depend on these for later projects, so they should come first.
 add_subdirectory(External)
 
-if(NOT TARGET SPIRV-Tools-opt)
-    set(ENABLE_OPT OFF)
-endif()
-
 if(ENABLE_OPT)
     message(STATUS "optimizer enabled")
     add_definitions(-DENABLE_OPT=1)
