--- glslang-15.1.0/StandAlone/CMakeLists.txt.orig	2025-03-23 13:58:35.622140918 +0100
+++ glslang-15.1.0/StandAlone/CMakeLists.txt	2025-03-23 14:08:21.628966247 +0100
@@ -109,7 +109,7 @@ if(GLSLANG_ENABLE_INSTALL)
         message(STATUS \"Installing (${link_method}): \$<TARGET_FILE_NAME:glslang-standalone> -> \$ENV{DESTDIR}\${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_BINDIR}\")
         execute_process(
             COMMAND \"\${CMAKE_COMMAND}\" -E ${link_method} [=[\$<TARGET_FILE_NAME:glslang-standalone>]=] [=[${legacy_glslang_name}]=]
-            WORKING_DIRECTORY \"\$ENV{DESTDIR}\${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_BINDIR}\"
+	    WORKING_DIRECTORY \"\$ENV{DESTDIR}${CMAKE_INSTALL_FULL_BINDIR}\"
         )
     ")
 
