diff -dur -x '*~' glslang-0967748fbce0773625dcba0f1185e1dd79092c0d.orig/glslang/MachineIndependent/intermOut.cpp glslang-0967748fbce0773625dcba0f1185e1dd79092c0d/glslang/MachineIndependent/intermOut.cpp
--- glslang-0967748fbce0773625dcba0f1185e1dd79092c0d.orig/glslang/MachineIndependent/intermOut.cpp	2016-02-19 20:21:50.000000000 +0100
+++ glslang-0967748fbce0773625dcba0f1185e1dd79092c0d/glslang/MachineIndependent/intermOut.cpp	2016-02-23 14:23:01.000000000 +0100
@@ -48,10 +48,8 @@
 bool is_positive_infinity(double x) {
 #ifdef _MSC_VER
   return _fpclass(x) == _FPCLASS_PINF;
-#elif defined __ANDROID__
-  return std::isinf(x) && (x >= 0);
 #else
-  return isinf(x) && (x >= 0);
+  return std::isinf(x) && (x >= 0);
 #endif
 }
 
