diff -dur glslang-4678ca9dacfec7a084dbc69bbe568bdad6889f1b.orig/Test/runtests glslang-4678ca9dacfec7a084dbc69bbe568bdad6889f1b/Test/runtests
--- glslang-4678ca9dacfec7a084dbc69bbe568bdad6889f1b.orig/Test/runtests	2016-05-13 17:33:42.000000000 +0200
+++ glslang-4678ca9dacfec7a084dbc69bbe568bdad6889f1b/Test/runtests	2016-07-14 10:00:43.000000000 +0200
@@ -1,8 +1,10 @@
 #!/usr/bin/env bash
 
+export LD_LIBRARY_PATH=../build/StandAlone
+
 TARGETDIR=localResults
 BASEDIR=baseResults
-EXE=../build/install/bin/glslangValidator
+EXE=../build/StandAlone/glslangValidator
 HASERROR=0
 mkdir -p localResults
 
